Mlem/App/Views/Shared/Search/SearchView+Views.swift

```diff
@@ -144,10 +160,28 @@ extension SearchView {
     @ViewBuilder
     var recentSearchesHeader: some View {
         HStack {
-            Text("Recently Searched")
-                .foregroundStyle(palette.primary)
+            if editingRecentSearches {
+                Button("Cancel") {
```

Comment:
Hmm also maybe this should be "Done". "Cancel" makes me think that it'll revert the changes I've just made, which it doesn't do.
```suggestion
                Button("Done") {
```